projects/plugins/social/src/class-note.php

```diff
@@ -196,8 +196,20 @@ public function maybe_flush_rewrite_rules( $force = false ) {
 	 * @param array $post_id The Post ID.
 	 */
 	public function override_empty_title( $title, $post_id ) {
-		if ( get_post_type( $post_id ) === self::JETPACK_SOCIAL_NOTE_CPT ) {
-			return wp_trim_words( get_the_excerpt(), 10 );
+		$post = get_post( $post_id );
+
+		if ( isset( $post ) && get_post_type( $post_id ) === self::JETPACK_SOCIAL_NOTE_CPT ) {
+			$post_date = get_the_date( 'd M Y', $post );
+			$post_time = get_the_time( 'H:i', $post );
+
+			/**
+			 * Filters the title for Social Notes.
+			 *
+			 * @param string   $new_title The generated title (e.g., 'Daily note 29 Mar 2025 03:32').
+			 * @param \WP_Post $post      The current post object.
+			 */
+			$label = apply_filters( 'jetpack_social_note_title_label', 'Daily note', $post );
+			return sprintf( '%s - %s %s', $label, $post_date, $post_time );
```

Comment:
A few issues here:
- We are not filtering the date and time strings here but the comment mentions them being a part of the label.
- We need to translate that label - `Daily note`
- We should also filter the whole title instead of just the label to make it easier for people to customize it.

So, I would suggest

```suggestion
			$label = __( 'Daily note', 'jetpack-social' );

			/**
			 * Filters the default label for Social Notes.
			 *
			 * @param string   $label The default label.
			 * @param \WP_Post $post  The current post object.
			 */
			$label = apply_filters( 'jetpack_social_notes_default_label', $label, $post );

			$title = sprintf( '%s - %s %s', $label, $post_date, $post_time );

			/**
			 * Filters the title for Social Notes.
			 *
			 * @param string   $title The generated title (e.g. 'Daily note - 29 Mar 2025 03:32 pm').
			 * @param \WP_Post $post  The current post object.
			 * @param string   $label The default label for the note.
			 */
			return apply_filters( 'jetpack_social_notes_default_title', $title, $post, $label );
```